prac_09/band.py

```diff
@@ -0,0 +1,24 @@
+"""Cp1404 Band class"""
+
+
+class Band:
+    def __init__(self, name):
+        """Initialize a Band with a name and an empty list of musicians."""
+        self.name = name
```

Comment:
I see that you give the variable names in classes to be generalized. Insead of 'band_name', you just used 'name'. While some object variable names can be paired with this (eg, 'band.name'). From there you can figure out that the band is getting its name. Sometimes you'll probably have object names that isn't pairable like that (eg, metal_necks.name). From there, you don't know what that object's class is. It is getting a name but from which class? By using band_name for class variable, you can tell which class is this from (eg, metal_necks.band_name)